west.yml

```diff
@@ -212,6 +214,12 @@ manifest:
       path: modules/hal/quicklogic
       groups:
         - hal
+    - name: hal_realtek
```

Comment:
module commit needs to be before everything else, can't add code that uses a module before the module is present

Reply:
we have issue a HAL module request: [hal_realtek](https://github.com/zephyrproject-rtos/zephyr/issues/78084)